### PR TITLE
10983: Add Scorecard alias data element to version data (db specfic pr)

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to add the Scorecard alias data element to the API pull request in GIDS so that the data can be included in version generation for search. This PR  includes only the `db` specific changes necessary for 10983.

The full PR can be found here: https://github.com/department-of-veterans-affairs/gibct-data-service/pull/695

Originating Issue: [10983](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10983)

## Testing done
CI passes locally, manual checks of columns in database as well.

## Screenshots
Screen Shot 2020-08-05 at 1 03 11 PM

## Acceptance criteria
- [x] The alias data element is retrieved and stored by the API Fetch request within GIDS.
- [x] The alias data element is included in the version generation.
- [x] The comparison tool search functionality includes the scorecard data element when a search is executed.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: db/migrate/20200804170036_add_alias_to_institutions.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddAliasToInstitutions < ActiveRecord::Migration[5.2]
```
Comment (zurbergram) [inline db/migrate/20200804170036_add_alias_to_institutions.rb:1]: This is not needed

File: db/migrate/20200804170056_add_alias_to_institutions_archives.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddAliasToInstitutionsArchives < ActiveRecord::Migration[5.2]
```
Comment (zurbergram) [inline db/migrate/20200804170056_add_alias_to_institutions_archives.rb:1]: This is not needed

**Review by zurbergram — CHANGES_REQUESTED** [review]
Remove not needed database changes

File: db/migrate/20200804170036_add_alias_to_institutions.rb (hunk 1 above)
Comment (elder-joseph) [reply db/migrate/20200804170036_add_alias_to_institutions.rb:1]: I can pull those out, wasn't sure if we wanted it to be stored under `alias` as it comes in from scorecard or `ialias` as it currently is

Comment (zurbergram) [reply db/migrate/20200804170036_add_alias_to_institutions.rb:1]: let's continue to use the existing column as `ialias` probably is just "institution alias"

also this reduces the code changes and database changes

**Review by zurbergram — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]

**Review by f1337 — APPROVED** [review]
Looks good to me! :thumbsup:
Shaun The Sheep giving thumbs up and smiling

**Pull request merged**